.github/actions/get-concurrency-group/README.md

```diff
@@ -0,0 +1,17 @@
+# Get the concurrency group for the running workflow
+
+This action is intended for getting the concurrency group for the running 
+workflow. This group name can be used with the `concurrency` feature to cancel 
+outdated workflow runs. See for more [details](https://git.io/J1XHy).
```

Comment:
Obscure links are discouraged. I suggests to use reference style links instead:

```markdown
Blah blah [blah][1]. Blah [blah][blah].

[1]: https://blah.blah/blah
[blah]: https://blahblah.blah
```

Reply:
Not relevant anymore when using the fake ternary operator.